doc/scripts/inputs.py

```diff
@@ -85,8 +85,8 @@
 ysoma_morph3 = representation.make_morph(tmp)
 
 tmp = [
-    [[Segment((0.0, 0.0, 1.0), (2.0, 0.0,  1.0), 1)], [Segment((-3.0, 0.0, 0.7), (0.0, 0.0, 1.0), 2)], [Segment((2.0, 0.0, 1.0), (20.0, 0.0, 1.0), 3)]],
-]
+    [[Segment((0.0, 0.0, 1.0), (2.0, 0.0, 1.0), 1), Segment((2.0, 0.0, 1.0), (20.0, 0.0, 1.0), 3)]],
```

Comment:
So, `inputs.py` is generated by `gen-images.py`. The proper procedure is to modify `gen-images.py`, use it to regenrate `inputs.py` and then push the changes to both files?

Reply:
Yes. I would vote we move to generating it on the fly soon.

Reply:
I decided to fix that as well.

Reply:
Thanks :)